Lib/defcon/objects/glyph.py

```diff
@@ -1161,6 +1168,88 @@ def _libContentChanged(self, notification):
         self.postNotification(notification="Glyph.LibChanged")
         self.dirty = True
 
+    # -----------------------------
+    # Serialization/Deserialization
+    # -----------------------------
+
+
+    def getDataForSerialization(self, **kwargs):
+        from functools import partial
+
+        simple_get = partial(getattr, self)
+        serialize = lambda item: item.getDataForSerialization()
+        serialized_get = lambda key: serialize(simple_get(key))
+        serialized_list_get = lambda key: [serialize(item) for item in simple_get(key)]
+
+        getters = [
+            ('name', simple_get),
+            ('unicodes', simple_get),
+            ('width', simple_get),
+            ('height', simple_get),
+            ('note', simple_get),
+            ('components', serialized_list_get),
+            ('anchors', serialized_list_get),
+            ('guidelines', serialized_list_get),
+            ('image', serialized_get),
+            ('lib', serialized_get)
+        ]
+
+        if self._shallowLoadedContours is not None:
+            getters.append( ('_shallowLoadedContours', simple_get) )
+        else:
+            getters.append( ('_contours', serialized_list_get) )
+
+        return self._serialize(getters, **kwargs)
+
+    def setDataFromSerialization(self, data):
+        from functools import partial, wraps
```

Comment:
I'm not sure. Maybe I used it at some point in development and then forgot to remove it again.